examples/components/singlepageviewer.js

```diff
@@ -36,16 +36,16 @@ var SEARCH_FOR = ''; // try 'Mozilla';
 var container = document.getElementById('viewerContainer');
 
 // (Optionally) enable hyperlinks within PDF files.
-var pdfLinkService = new PDFJS.PDFLinkService();
+var pdfLinkService = new pdfjsDistWebPdfViewer.PDFLinkService();
 
-var pdfSinglePageViewer = new PDFJS.PDFSinglePageViewer({
+var pdfSinglePageViewer = new pdfjsDistWebPdfViewer.PDFSinglePageViewer({
   container: container,
   linkService: pdfLinkService,
 });
 pdfLinkService.setViewer(pdfSinglePageViewer);
 
 // (Optionally) enable find controller.
-var pdfFindController = new PDFJS.PDFFindController({
+var pdfFindController = new pdfjsDistWebPdfViewer.PDFFindController({
```

Comment:
For consistency with the other examples, could you add a trailing comma on the next line?

Reply:
Fixed.